README.md

```diff
@@ -39,6 +41,51 @@ anonymize(data);
 // }
 ```
 
+#### Example using serializers
+
+```js
+const { anonymizer } = require('@uphold/anonymizer');
+const whitelist = ['foo.key', 'foo.depth.*', 'bar.*', 'toAnonymize.baz'];
+const blacklist = ['foo.depth.innerBlacklist'];
+const serializers = [
+  { path: 'foo.key', serializer: () => 'biz' },
+  { path: 'toAnonymize', serializer: () => ({ baz: 'baz' }) }
+]
+const anonymize = anonymizer({ blacklist, whitelist });
+
+const data = {
+  foo: { key: 'public', another: 'bar', depth: { bar: 10, innerBlacklist: 11 } },
+  bar: { foo: 1, bar: 2 },
+  toAnonymize: {}
+};
+
+// {
+//   foo: {
+//     key: 'biz',
+//     another: '--REDACTED--',
+//     depth: { bar: 10, innerBlacklist: '--REDACTED--' }
+//   },
+//   bar: { foo: 1, bar: 2 },
+//   toAnonymize: { baz: 'baz' }
+// }
+```
+
+### Default serializers
+
+The introduction of serializers also added the possibility of using serializer functions exported by our module. The list of default serializers is presented below:
+- error
+
+#### Example
+
+```js
+const { anonymizer, defaultSerializers } = require('@uphold/anonymizer');
+const serializers = [
+  { path: 'foo', serializer: defaultSerializers.error }
+];
+
+const anonymize = anonymizer({ whitelist }, { serializers });
```

Comment:
Complete the example with the anonymize instruction and the outcome?

Reply:
Done